boards/shields/atmel_rf2xx/atmel_rf2xx_arduino.overlay

```diff
@@ -6,7 +6,7 @@
 
 / {
 	chosen {
-		zephyr,ieee802154 = &ieee802154;
+		zephyr,ieee802154_atmel_rf2xx_arduino = &ieee802154_atmel_rf2xx_arduino;
```

Comment:
I didn`t understand how this will bind with samples, for instance:

https://github.com/zephyrproject-rtos/zephyr/blob/1b98db232ed32ce52d7ca9e64a7bfb44c3da0055/samples/net/wpanusb/src/wpanusb.c#L40-L41

Should samples have a overlay file?

Reply:
ok, my bad. `zephyr,ieee802154` should not have been changed. I'll fix it